/rosky/wall_following:
  ros__parameters:
    linear_speed_limit : 0.05
    angular_speed_limit : 0.5

    # PD controller
    pid_standard : 2.0        # front distance limit = motor_length * pid_standard, right distance limit = motor_width * pid_standard
    pid_thres_hold_min : 0.05
    pid_thres_hold_max : 0.07 # thres_hold_min + 0.02
    pid_kp : -3.0
    pid_kd : -5.0